packages/experimental/Text/src/Text.tsx

```diff
@@ -12,9 +12,10 @@ import {
   compressible,
   patchTokens,
 } from '@fluentui-react-native/framework';
-import { Text as RNText, ColorValue } from 'react-native';
+import { ColorValue } from 'react-native';
 import { ITextProps } from '@fluentui-react-native/adapters';
 import React from 'react';
+import { Text as RNText } from './TextComponent';
```

Comment:
What was wrong with the previous approach? Just that we needed to import from 'react-native' for most platforms, but from '@office-iss/react-native-win32' for others? Would we need to do similar imports from 'react-native-windows' / 'react-native-macos' ?